modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -1,3 +1,3 @@
-.cke_inner .cke_top {
+.cke_float .cke_top {
```

Comment:
I don't mind this solution, but this is not restricting the fix to BalloonEditor, but to every floating toolbar. Developers could be using CKEditor through the JSP tag or directly via the `Editor` or `ClassicEditor` React components and setting mode to `inline` to get floating toolbars. This change would still target those.

Reply:
Yes you're right, as soon as we change the editor's mode to `inline` these styles will be applied (and that's not good enough)

Since these are "global" styles, I was wondering if using "styled components" would make sense, i.e. something like [this](https://scalablecss.com/styled-components-global-styles/), and make sure this only happens for `BalloonEditor`, I'm going to have a look just to see what happens.

Reply:
Hey @julien, I think that's going way off course and into very unknown territory and this is blocking the GA1 release, so let's try to figure out a more immediate solution.

If we can't add a marker to the toolbar, let's either push this as is or revert the thing to have more time to think it through, but we need to provide a short-term solution.

Reply:
@jbalsas I've done that in [https://github.com/liferay-frontend/liferay-portal/commit/b83b82f873a3f23c921c5228e9e5bc676538fa57](https://github.com/liferay-frontend/liferay-portal/commit/b83b82f873a3f23c921c5228e9e5bc676538fa57) and removed my previous commit.
I'll update the pull request description right now.